PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py

```diff
@@ -22,6 +24,7 @@
 
 modifyPrimaryPhysicsObjects = cms.Sequence( electronMVAValueMapProducer *
                                             photonIDValueMapProducer * photonMVAValueMapProducer * 
+                                            electronRegressionValueMapProducer * photonRegressionValueMapProducer *
```

Comment:
This sequence isn't used in MiniAOD (but only in MiniAOD-from-MiniAOD,
which was never fully setup in 75X/76X).
That's why these differences are here.

On Mon, Aug 24, 2015 at 7:18 AM, Slava Krutelyov <notifications@github.com>
wrote:

> In
> PhysicsTools/PatAlgos/python/slimming/modifyPrimaryPhysicsObjects_cff.py
> <https://github.com/cms-sw/cmssw/pull/10693#discussion_r37719977>:
>
> > @@ -22,6 +24,7 @@
> >
> >  modifyPrimaryPhysicsObjects = cms.Sequence( electronMVAValueMapProducer *
> >                                              photonIDValueMapProducer * photonMVAValueMapProducer *
> > +                                            electronRegressionValueMapProducer * photonRegressionValueMapProducer *
>
> these are not in 75X/76X
> why needed here?
> Isn't the modification in egmGsfElectronIDSequence enough?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cms-sw/cmssw/pull/10693/files#r37719977>.
>